### ISSUE TITLE
Verify wallet login message isn't translated in brave://rewards settings page for few locales

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#12017]
## Description 

Verify wallet login message isn't translated in brave://rewards settings page for few locales
## Steps to Reproduce

   1. Set OS locale to Kannada
   2. Install brave 1.15.x
   3. Enable rewards
   4. Click on `Verify wallet` in brave://rewards settings page

## Actual result:

Verify wallet login message isn't translated in brave://rewards settings page for few locales
![image](https://user-images.githubusercontent.com/38657976/95214119-2e574380-080d-11eb-9a65-7fd092ef44cc.png)

## Expected result:
Verify wallet login message should be translated to OS locale

**1.14.x**
![image](https://user-images.githubusercontent.com/38657976/95218998-7fb60180-0812-11eb-86f2-478b01938243.png)

## Reproduces how often: 

Always
## Brave version (brave://version info)

Brave | 1.15.71 Chromium: 86.0.4240.72 (ಅಧಿಕೃತವಾಗಿ ನಿರ್ಮಿಸಿ) (64-ಬಿಟ್)
-- | --
ಪರಿಷ್ಕರಣೆ | 581582174c512f44f44fd1aea340471f54b2365f-refs/branch-heads/4240@{#1134}
OS | Windows 10 OS Version 1809 (Build 17763.1397)

## Version/Channel Information:

- Can you reproduce this issue with the current release? No
- Can you reproduce this issue with the beta channel? Not sure
- Can you reproduce this issue with the nightly channel? Not sure

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/legacy_qa @rebron

**Comment by kjozwiak** [comment]
Closing and moved into https://github.com/brave/brave-browser/milestone/135 as https://github.com/brave/brave-core/pull/6788 has been merged into `1.15.x` 👍

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.15.72 Chromium: 86.0.4240.75 (Offizieller Build) (x86_64)
-- | --
Überarbeitung | c69c33933bfc72a159aceb4aeca939eb0087416c-refs/branch-heads/4240@{#1149}
Betriebssystem | macOS Version 10.14.6 (Build 18G3020)
```

Verified STR from description for both French and German:

**`French`**
<img width="410" alt="12017 French" src="https://user-images.githubusercontent.com/28145373/95338964-8039b100-0881-11eb-97c1-5825fdeb663a.png">

**`German`**
<img width="398" alt="12017 German" src="https://user-images.githubusercontent.com/28145373/95338975-829c0b00-0881-11eb-87a9-4b64e2b57b44.png">

------------
